The executables are placed under CMAKE_CURRENT_BINARY_DIR and
looking for them under CONFIGURATION (Release) won't work.
$OpenBSD$
--- googletest/cmake/internal_utils.cmake.orig
+++ googletest/cmake/internal_utils.cmake
@@ -243,7 +243,7 @@ function(py_test name)
       add_test(
         NAME ${name}
         COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/test/${name}.py
-            --build_dir=${CMAKE_CURRENT_BINARY_DIR}/$<CONFIGURATION>)
+            --build_dir=${CMAKE_CURRENT_BINARY_DIR}/)
     else (${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} GREATER 3.1)
       add_test(
         ${name}
